development-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+
+services:
+  web:
+    container_name: web
+    build: .
+    environment:
+      - DATABASE_URL=postgresql://postgres:postgres@db:5432/daily-code?schema=public
```

Comment:
@fasal26 Please use .env for DATABASE_URL. This would help avoid inconsistencies and it is secure.

Eg: 
    environment:
      - DATABASE_URL=${DATABASE_URL}

Reply:
On it